string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,23 @@
+/* exported capitalizeWord */
+
+/*
+  - Create variable for new string
+  - Loop through word and make all letters lowercase
+  - Make word into all lowercase
+  - if (word lowercased === javascript) then capitalize j and s index
+  - Then create new variable which upper cases first letter and adds lowercased
+  - Return uppercase variable
+*/
+
+function capitalizeWord(word) {
+  var lowerCaseWord = word.toLowerCase();
+  var newWord = '';
+  for (var i = 0; i < lowerCaseWord.length; i++) {
+    newWord += lowerCaseWord[i];
+  }
```

Comment:
Try removing the loop completely.

Reply:
Removed loop from function

<img width="1148" alt="Screen Shot 2022-02-09 at 12 40 14 PM" src="https://user-images.githubusercontent.com/87846307/153286519-b541e8ec-ac99-4f0a-84c4-9f7b258864ab.png">